client/themes/zenburn.css

```diff
@@ -18,6 +18,10 @@ BORDERS #333333
 QUIT #bc6c4c
 */
 
+body {
+	background: #2b2b2b;
+}
+
```

Comment:
@JocelynDelalande Yes it is, this is what it looks like without that:

[![screenshot_2015-10-07-05-14-22](https://cloud.githubusercontent.com/assets/1300395/10334848/90f95ed4-6cb2-11e5-87b6-7c86fa4a4278.png)](https://cloud.githubusercontent.com/assets/1300395/10334821/60283aa0-6cb2-11e5-8564-4059b9cf26ec.png)